oak_attestation_verification/tests/verifier_tests.rs

```diff
@@ -70,8 +120,11 @@ fn verify_succeeds() {
     let endorsements = create_endorsements();
     let reference_values = create_reference_values();
 
-    let r = verify(&evidence, &endorsements, &reference_values);
+    let r = verify(NOW_UTC_MILLIS, &evidence, &endorsements, &reference_values);
 
+    println!("======================================");
```

Comment:
It seems these logs should be printed to stderr instead (`eprintln`) (also elsewhere)

Reply:
Done. This will likely go after wrapping up development around here.